javaee/api/src/main/java/org/jboss/forge/addon/javaee/servlet/SecurityConstraintDescriptor.java

```diff
@@ -0,0 +1,177 @@
+package org.jboss.forge.addon.javaee.servlet;
+
+import org.jboss.forge.addon.javaee.security.TransportGuarantee;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Descriptor of the security constraint element part of the web.xml
+ *
+ * @author <a href="mailto:ivan.st.ivanov@gmail.com">Ivan St. Ivanov</a>
+ */
+public class SecurityConstraintDescriptor
```

Comment:
Aha :)

Well, in this API we lack most of the common classes concerning web.xml. In https://github.com/forge/javaee-descriptors/tree/master/api/src/main/java/org/jboss/shrinkwrap/descriptor/api/webapp there is only one class.

You think we should refactor the project?